### PR TITLE
feature: zios-9970 [Tablet] Gallery view is opened full screen instead of being opened inside a popover

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

Display camera roll image picker with popover style on iPad regular mode.

### CODE REVIEW

**Comment by mikeger** [comment]
@billypchan conflict on the project file 🔥

File: Wire-iOS/Sources/UserInterface/Camera/CameraBottomToolsViewController.m [hunk 1]
```diff
@@ -187,20 +187,11 @@ - (IBAction)closeCamera:(id)sender
 
 - (IBAction)openPhotoLibrary:(id)sender
 {
-    UIImagePickerController *picker = [[UIImagePickerController alloc] init];
+    UIImagePickerController *picker = [UIImagePickerController popoverForIPadRegularWithSourceRect:CGRectInset(self.libraryButton.bounds, 4, 4) sourceView:self.libraryButton presentViewController:self sourceType:UIImagePickerControllerSourceTypePhotoLibrary];
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Camera/CameraBottomToolsViewController.m]: I think that this method's signature is too long, may be it is better to create the immutable configuration object that would hold all those parameters? Like `ImagePickerPresentationContext`, or something similar?

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/UIImagePickerController+iPadPopover.swift [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+extension UIImagePickerController {
+    @objc class func popoverForIPadRegular(sourceRect: CGRect,
+                                           sourceView: UIView,
+                                           presentViewController: UIViewController,
+                                           sourceType: UIImagePickerControllerSourceType) -> UIImagePickerController {
+        let picker = UIImagePickerController()
+        picker.sourceType = sourceType
+
+        if presentViewController.isIPadRegular(device: UIDevice.current) {
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/InputBar/UIImagePickerController+iPadPopover.swift]: I think that the method `isIPadRegular` is not relevant to `UIViewController`, would be significantly better to have it on something relevant to it, for example on `UIDevice`.

Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Conversation/InputBar/UIImagePickerController+iPadPopover.swift]: @mikeger I will rewrite it like: 

if presentViewController.traitCollection.horizontalSizeClass.isIPadRegular(device: UIDevice.current)

Comment (mikeger) [reply Wire-iOS/Sources/UserInterface/Conversation/InputBar/UIImagePickerController+iPadPopover.swift]: Ah, I see now that it checks the trait collection, then I guess it is fine on UIViewController.

**Pull request merged**